tests/scadnano_tests.py

```diff
@@ -919,10 +928,11 @@ def test_export_design_with_helix_group(self):
         }
 
         design = sc.Design(helices=helices, groups=groups, strands=[])
-        design.write_scadnano_file(directory=self.input_path,
-                                   filename=f'test_export_design_with_helix_group.{self.ext}')
-        design.export_cadnano_v2(directory=self.output_path,
-                                 filename='test_export_design_with_helix_group.json')
+        # design.write_scadnano_file(directory=self.input_path,

```

Comment:
Can we remove the commented out lines? I don't think it's necessary to preserve the old code here or in other unit tests, once we're sure this new code is properly testing the methods.

Reply:
I was thinking it might be nice to have these here so that if these tests are failing, we can easily temporarily uncomment these lines, run the tests to generate the file, and then open the file on scadnano/cadnano. Do you think this is enough of a reason to keep the commented out lines?

Reply:
> I was thinking it might be nice to have these here so that if these tests are failing, we can easily temporarily uncomment these lines, run the tests to generate the file, and then open the file on scadnano/cadnano. Do you think this is enough of a reason to keep the commented out lines?

Okay, then I'd put that explanation before the commented lines so we remember to keep them there in the future and don't think it's just old stuff that should be removed.

Reply:
Sounds good, I'll add these to the comments